### DISCUSSION TITLE
Integrar meios de Pagamento com Chamilo no Brasil - Pagseguro ou Mercado Pago

### DISCUSSION THREAD

**Discussion by pridecommerce** [chamilo/chamilo-lms#6007] [General] [answered]
Preciso de ajuda para isso, posso até pagar para isso, preciso que depois de um pagamento concluído, se crie um usuário novo e envie para o aluno e pra nós um e-mail de confirmação.

**Comment by ywarnier** [comment, marked as answer]
Hi @pridecommerce 

Sorry for the late answer and sorry for using English.
You can ask one of the official providers of Chamilo to develop this for you. See https://chamilo.org/providers or send them an e-mail directly at providers @ chamilo dot org.
Otherwise, someone might be able to help you out by just extending the current stripe or paypal methods.

The creation of users and e-mailing is already included in the `buycourses` plugin. It just needs the implementation of the Mercado Pago gateway. This can be complex, but at least the rest is already there.
